internal/cloud/atlas/groups.go

```diff
@@ -39,5 +50,16 @@ func (c *client) Groups() ([]Group, error) {
 	if err := json.NewDecoder(res.Body).Decode(&groupRes); err != nil {
 		return nil, err
 	}
-	return groupRes.Results, nil
+
+	allGroups := groupRes.Results
+	for _, link := range groupRes.Links {
+		if link.Rel == "next" {
```

Comment:
[micro-opt] we end up iterating through _all_ links here no matter what... I might suggest doing something like:

```go
for _, link := range groupRes.Links {
  if link.Rel != "next" {
    continue
  }
  nextGroups, err := ...
  // ...
  allGroups = append(allGroups, nextGroups...)
  break
}
```

---

also, can we move `"next"` to a const here: `linkRelNext = "next"`?